src/createAsyncStoragePersisor-experimental/index.js

```diff
@@ -0,0 +1,45 @@
+import AsyncStorage from '@react-native-community/async-storage';
+
+function throttle(func, wait) {
+  if (wait === 0) {
+    wait = 100;
+  }
+
+  var timer = null;
+  return function () {
+    for (
+      var _len = arguments.length, args = new Array(_len), _key = 0;
+      _key < _len;
+      _key++
+    ) {
+      args[_key] = arguments[_key];
+    }
+
+    if (timer === null) {
+      timer = setTimeout(function () {
+        func.apply(0, args);
+        timer = null;
+      }, wait);
+    }
+  };
+}
+
+export const asyncStoragePersistor = ({asyncStorageKey, throttleTime}) => {
+  return {
+    persistClient: throttle(function (persistedClient) {
```

Comment:
I'm not sure whether we should use throttle here because `setItem` is async so probably we want to return this promise as well :)

Reply:
73a20c4b